### ISSUE TITLE
error: Invalid value for '--update-map-size <update-map-size>': invalid digit found in string

### ISSUE THREAD

**Issue by NOVA-ME** [meilisearch/MeiliSearch#650]
I clone the master branch and compile it by myself,when I start the meilisearch,it cause an error,I check the default value is not a string,I'm confuse about it.
```bash
➜ cargo build --release
➜ ./meilisearch
error: Invalid value for '--update-map-size <update-map-size>': invalid digit found in string
```
The source code located at ```meilisearch_core::DatabaseOptions::default().update_map_size ```
```rust
impl Default for DatabaseOptions {
    fn default() -> DatabaseOptions {
        DatabaseOptions {
            main_map_size: 100 * 1024 * 1024 * 1024, // 100GB
            update_map_size: 100 * 1024 * 1024 * 1024 // 100GB
        }
    }
}
```

**Comment by Kerollmops** [comment]
Thank you for the report this has already been fixed by #647.

**Comment by NOVA-ME** [comment]
OK,thank you so much.
